console/src/app/modules/policies/notification-sms-provider/dialog-add-sms-provider/dialog-add-sms-provider.component.html

```diff
@@ -18,6 +18,11 @@ <h1 mat-dialog-title class="title">
       <input cnslInput name="senderNumber" formControlName="senderNumber" />
     </cnsl-form-field>
 
+    <cnsl-form-field class="sms-form-field" label="Verification Service Sid">
+      <cnsl-label>{{ 'SETTING.SMS.TWILIO.VERIFYSERVICESID' | translate }}</cnsl-label>
+      <input cnslInput name="verifyServiceSid" formControlName="verifyServiceSid" />
+    </cnsl-form-field>
```

Comment:
So I'm thinking the text might say something along the lines of "A Sender Number OR a Verification Service ID is required" since we only will use one and the verification number overrides the sender number for all current use-cases. 

Having both could be useful for future sms notifications that aren't OTP. Thoughts?